components/edit-collective/EditHostInvoice.js

```diff
@@ -12,16 +12,24 @@ import Container from '../Container';
 import MessageBox from '../MessageBox';
 import StyledButton from '../StyledButton';
 import StyledInput from '../StyledInput';
+import StyledTextarea from '../StyledTextarea';
 import { H3, P } from '../Text';
 import { updateSettingsMutation } from './mutations';
 
 const EditHostInvoice = ({ collective }) => {
+  // For invoice Title
   const defaultValue = get(collective.settings, 'invoiceTitle');
   const [setSettings, { loading, error, data }] = useMutation(updateSettingsMutation);
   const [value, setValue] = React.useState(defaultValue);
   const isTouched = value !== defaultValue;
   const isSaved = get(data, 'editCollective.settings.invoiceTitle') === value;
 
+  // For invoice extra info
+  const defaultInfo = get(collective.settings, 'extraInfo');
+  const [info, setInfo] = React.useState(defaultInfo);
+  const isTouch = info !== defaultInfo;
+  const isSave = get(data, 'editCollective.settings.extraInfo') === info;
```

Comment:
If you need different names because `isTouched` is already taken, use something explicit like:

```suggestion
  const infoIsTouched = info !== defaultInfo;
  const infoIsSaved = get(data, 'editCollective.settings.extraInfo') === info;
```